io.elementary.Sdk.json

```diff
@@ -183,6 +209,28 @@
                     "tag": "1.2.0"
                 }
             ]
+        },
+        {
+            "name" : "platform-settings",
+            "buildsystem" : "simple",
+            "modules": [
+                {
+                    "name" : "default-gtk-settings",
+                    "buildsystem" : "simple",
+                    "sources" : [
+                        {
+                            "type": "git",
+                            "url": "https://github.com/elementary/default-settings.git"
+                        }
+                    ],
+                    "build-commands": [
+                        "install -Dm644 gtk/settings.ini /etc/gtk-3.0/settings.ini"
+                    ]
+                }
+            ],
+            "build-commands": [
+                "ln -S /usr/share/themes/io.elementary.stylesheet.blueberry /usr/share/themes/elementary"
```

Comment:
Where does this help, an app will either be built against eos5 or eos6 right? at which point will it a eos6 app be expecting `elementary` as the name?

Reply:
If the user is on eOS 5 then its theme name will be elementary and might be exported, that helps with the fallback

Reply:
do we want make the same with Adwaita? or is better to remove them from the Sdk and let the fallback manage it?